gridstatus/ercot_api/ercot_api.py

```diff
@@ -295,6 +295,10 @@ def _handle_hourly_wind_report(self, data, verbose=False):
 
         data = Ercot()._rename_hourly_wind_or_solar_report(data)
 
+        # Add HSL SYSTEM WIDE if it is not in the data (older data may not have it)
```

Comment:
do we want to put this in `_rename_hourly_wind_or_solar_report` so we've centralized all this standardization logic?

Reply:
Yes, good idea, updated.